embedded_sass.proto

```diff
@@ -831,6 +831,57 @@ message Value {
     map<string, Value> keywords = 4;
   }
 
+  // A SassScript calculation value. The compiler must send fully [simplified]
+  // calculations, meaning that simplifying it again will produce the same
+  // calculation. The host is not required to simplify calculations.
+  //
+  // [simplified]: https://github.com/sass/sass/tree/main/spec/types/calculation.md#simplifying-a-calculation
+  //
+  // The compiler must simplify any calculations it receives from the host
+  // before returning them from a function. If this simplification produces an
+  // error, it should be treated as though the function call threw that error.
+  // It should *not* be treated as a protocol error.
+  message Calculation {
+    // The calculation's name. Mandatory. The host may only set this to names
+    // that the Sass specification uses to create calculations.
+    string name = 1;
+
+    // The calculation's arguments. Mandatory. The host must use exaclty the
+    // number of arguments used by the Sass specification for calculations with
+    // the given `name`.
+    repeated CalculationValue argument = 2;
+
+    // A single component of a calculation expression.
+    message CalculationValue {
+      // The value of the component. Mandatory.
+      oneof value {
+        Number number = 1;
+
+        // An unquoted string, as from a function like `var()` or `env()`.
+        string string = 2;
+
+        // An unquoted string as created by interpolation for
+        // backwards-compatibility with older Sass syntax.
+        string interpolation = 3;
+
+        CalculationOperation operation = 4;
+        Calculation calculation = 5;
```

Comment:
Should these be documented too?

Reply:
I thought about it, but I decided not to on the same principle that the different value options aren't documented: there's really nothing to say other than "an operation" and "a calculation".